src/app/frontend/service/detail/actionbar.html

```diff
@@ -14,6 +14,13 @@
 limitations under the License.
 -->
 
+<md-button ng-click="$ctrl.handleScaleResourceDialog()">
+  <md-icon class="kd-actionbar-icon-button">linear_scale</md-icon>
+  <md-tooltip md-direction="bottom">
+    [[Edit number of pods|Tooltip for the 'scale' button on the action bar of a deployment details view.]]
+  </md-tooltip>
+  [[Scale|Tooltip for the 'scale' button on the action bar of a deployment details view.]]
+</md-button>
```

Comment:
How do you want to scale a service?

![image](https://user-images.githubusercontent.com/2823399/27778871-e16524c4-5fbe-11e7-87a3-93e20a5de218.png)